fele-client-service/cli/scripts/channel.js

```diff
@@ -1,22 +1,102 @@
-const fs = require('fs');
+/*const { checkIfNetworkExists, insertToDatabase } = require('../../utils/db')
+const { validateSchema } = require('../../utils/schema-validator')
 const path = require("path");
+const fs = require('fs');
+const USER_WORKSPACE = "../../../tmpworkspaceforuser/"
+const jsonSch = 'feleChannelSchema.json'
 
-function createChannel(networkName, channelName) {
-  var dir = "../../../chaincode/"+networkName+"/"+channelName
-  dir = path.join(__dirname, dir);
-  try {
-      if (!fs.existsSync(dir)) {
-        fs.mkdirSync(dir)
-        console.log('Directory created.')
-      } else {
-        console.log('Directory already exists.')
-      }
-  } catch (err) {
-    console.log(err)
-  }
+const createChannel = async (networkName, channelName, channelConfig) =>
+{
+  const database = "fele__"+networkName;
+  try{
+    const dbStatus = checkIfNetworkExists(database)
+    if(dbStatus){
+        console.log("Network "+ database+" exists")
+        if(channelConfig.includes(".json")) {
+          const fileName = USER_WORKSPACE+channelConfig;
+          channelConfig = require(fileName);
+          channelConfig = JSON.stringify(channelConfig)
+        } else {
+          channelConfig = channelConfig.replace(/\\/g,'')
+        }
+        if(jsonSch.includes(".json")) {
+          const jsonSchema = USER_WORKSPACE+jsonSch;
+          jsonSchema = require(fname);
+          jsonSchema = JSON.stringify(jsonSchema);
+        }
+        else{
+          jsonSch = jsonSch.replace(/\\/g,'')
+        }
+        const valschema = validateSchema(jsonSchema, channelConfig);
+        const isValid = valschema['errors'];
+        console.log(isValid);
+        if (isValid){
+          const cconfig = insertToDatabase(database, JSON.parse(channelConfig));
+          if (cconfig){
+            var dir = "../../../chaincode/"+networkName+"/"+ channelName
+            dir = path.join(__dirname, dir)
+            if (!fs.existsSync(dir)) {
+                fs.mkdirSync(dir)
+                console.log('Directory created.')
+                }
+            else {
+                console.log('Directory already exists.')
+                }
+            }
+       }
+        else{
+         console.error(valschema.message);
+        }
+    }
+     }catch(error) {
+         return false
+     }
+     return false
+ }
+
+ module.exports = {
+  createChannel
 }
+*/
+const { checkIfNetworkExists, insertToDatabase } = require('../../utils/db')
+const fs = require('fs');
+const path = require("path");
+const USER_WORKSPACE = "../../../tmpworkspaceforuser/"
+
+function createChannel(networkName, channelName, channelConfig)
```

Comment:
We already implemented the create channel functionality `client-api/scripts`. You can make use of that function.
That way we can keep our code DRY.
You can add additional functionality for CLI here. Like checking the channel config parameter for JSON Object or file name. 
If it is file name you can import the json from tmpworkspace and frame your inputs and call the function which is we have defined.
Any common code for CLI/REST API should be written only once